src/plugins/janus_streaming.c

```diff
@@ -10348,4 +10353,4 @@ static void *janus_streaming_helper_thread(void *data) {
 	janus_refcount_decrease(&mp->ref);
 	g_thread_unref(g_thread_self());
 	return NULL;
-}
+}
```

Comment:
Please don't remove empty lines at the end of files

Reply:
sry, reverted back